apps/frontend/app/(website)/studio/src/components/CodemodButton/ProgressBar.tsx

```diff
@@ -0,0 +1,32 @@
+import Text from "@studio/components/Text";
+import { Progress } from "@studio/components/ui/progress";
+import type { useCodemodExecution } from "@studio/hooks/useCodemodExecution";
+
+export type ProgressBarProps = {
+  codemodRunStatus: ReturnType<typeof useCodemodExecution>["codemodRunStatus"];
+};
+export const ProgressBar = ({ codemodRunStatus }: ProgressBarProps) => {
+  const progressInfo =
```

Comment:
something is smelly here. `codemodRunStatus.progressInfo` should be  enough. Why do we need to check if codemodRunStatus?.status is `progress`. If it is not in progress then `codemodRunStatus.progressInfo` is null isn't it? 🤔